dffbackend/BusinessLogic/Signatures/Agents/SignaturesAgent.cs

```diff
@@ -79,6 +79,7 @@ public async Task<CheckDuplicatesResponseDto> CheckSignatureSetForDuplicates(str
     public async Task StoreSignatureSet(string requesterId, Signature signatureSet)
     {
         signatureSet.FactoringCompanyId = new Guid(requesterId);
+        signatureSet.CreationDate = DateTime.Now;
```

Comment:
try not to mix chnges from different branches into a single PR. Next time, just branch from main so that only relevant changes for the story are on the branch.

Reply:
Okay, I understand what I did wrong there.